packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -54,18 +52,20 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
     const ImageType = (imageLoadError && imageErrorAs) || Image;
 
     // eslint-disable-next-line deprecation/deprecation
-    const ariaLabel = this.props['aria-label'] || this.props.ariaLabel;
-    const containerProps = ariaLabel
+    const accessibleName = imageProps.alt || this.props['aria-label'] || this.props.ariaLabel;
```

Comment:
I think it would be clearer to do this (plus the suggestion a couple lines down, and adding back aria-label to the native props exclude list above)
```suggestion
    const ariaLabel = this.props['aria-label'] || this.props.ariaLabel;
    const accessibleName = imageProps.alt || ariaLabel;
```